spec/models/user_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { create(:user) }
+
+  describe 'validations' do
+    it 'is valid with valid attributes' do
+      expect(subject).to be_valid
+    end
+
+    it 'is not valid without a name' do
+      subject.name = nil
+      expect(subject).not_to be_valid
+    end
+
+    it 'is not valid without an email' do
+      subject.email = nil
+      expect(subject).not_to be_valid
+    end
+
+    it 'is not valid without a valid phone number' do
+      subject.phone = nil
+      expect(subject).not_to be_valid
+    end
+
+    it 'is valid with a valid phone number' do
+      subject.phone = '+8801712345678'
+      expect(subject).to be_valid
+    end
```

Comment:
add a context for phone numbers and keep the phone number validation specs inside that context

Reply:
Done bhai.